packages/vector_graphics/lib/src/vector_graphics.dart

```diff
@@ -376,13 +371,17 @@ class _VectorGraphicWidgetState extends State<VectorGraphic> {
   }
 
   void _handleError(Object error, StackTrace? stackTrace) {
+    if (!mounted) {
+      return;
+    }
+
     setState(() {
       _error = error;
       _stackTrace = stackTrace;
     });
   }
 
-  void _loadAssetBytes() {
+  Future<void> _loadAssetBytes() async {
```

Comment:
I initially tried using a `catchError()` on the `_loadPicture()` call, but that still caused the written test to fail.

Since async work in a `void` method smells of https://dart.dev/tools/linter-rules/avoid_void_async I opted to change this to `Future<void>`, which did fix my test.